src/app_service/service/ens/utils.nim

```diff
@@ -87,6 +88,15 @@ proc buildTransaction*(
   else:
     result.txType = "0x00"
 
+proc buildTransactionDataDto*(gasUnits: int, suggestedFees: SuggestedFeesDto, addressFrom: string, chainId: int, contractAddress: string): TransactionDataDto =
+    if suggestedFees == nil:
+      error "Can't find suggested fees for chainId", chainId=chainId
+      return
+    return buildTransaction(parseAddress(addressFrom), 0.u256, $gasUnits,
+      if suggestedFees.eip1559Enabled: "" else: $suggestedFees.gasPrice, suggestedFees.eip1559Enabled,
```

Comment:
@endulab gas price should not depend on eip1559 and should not be empty.

Reply:
Why gas price is needed for eip1559 tx?